src/services/services.ts

```diff
@@ -2276,6 +2277,25 @@ namespace ts {
                             }
                         }
                     }
+                    else if (token === SyntaxKind.ImportKeyword) {
+                        token = scanner.scan();
+                        if (token === SyntaxKind.Identifier || isKeyword(token)) {
+                            token = scanner.scan();
+                            if (token === SyntaxKind.EqualsToken) {
+                                token = scanner.scan();
+                                if (token === SyntaxKind.RequireKeyword) {
+                                    token = scanner.scan();
+                                    if (token === SyntaxKind.OpenParenToken) {
+                                        token = scanner.scan();
+                                        if (token === SyntaxKind.StringLiteral) {
+                                            //  export import i = require("mod");
+                                            recordModuleName();

```

Comment:
It would be great if you had a helper function like

```TypeScript
function scanYieldsSequence(scanner: Scanner, seq: (SyntaxKind | ((kind: SyntaxKind) => boolean))[]) {
    for (let item of seq) {
        let passes = item instanceof Function ?
            item(scanner.scan()) :
            item === scanner.scan()
        if (!passes) {
            return false;
        }
    }
    
    return true;
}
```

but it might be overkill